src/main/index.js

```diff
@@ -1,16 +1,22 @@
 'use strict'
 
 import './globalSetting'
-import { app, Menu } from 'electron'
+import { app, ipcMain, Menu } from 'electron'
 import configureMenu, { dockMenu } from './menus'
 import createWindow, { windows } from './createWindow'
+import { isMarkdownFile } from './utils'
 // import { autoUpdater } from "electron-updater"
 
 let openFilesCache = []
 
 const onReady = () => {
-  if (process.platform === 'win32' && process.argv.length >= 2) {
-    openFilesCache = [process.argv[1]]
+  if (process.platform !== 'darwin' && process.argv.length >= 2) {
```

Comment:
This should be apply for OSX as well or why was it limited to windows?

Reply:
Because we already listen for `open-file` event in OS X system, but `open-file` event is only for MAC.

PLS refer https://electronjs.org/docs/api/app

Reply:
I don't know if it is the code or windows but this breaks open with.

Reply:
I am not sure if this works.

Reply:
Doesn't break anything. Compiled with #53.